Clarify that metrics can be used in query-level filters (#5733)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/ref-metrics-in-filters.md

        - name: accounts
          agg: sum
          expr: 1

>       create_metric: true

Reviewer: ```suggestion
          create_metric: true
```

>       create_metric: true

Reviewer: ```suggestion
           create_metric: true
```

Reviewer: @courtneyholcomb just moving this indentation under measures = hope that's ok!

>           create_metric: true

Reviewer: @courtneyholcomb just moving this indentation under measures = hope that's ok!

>            create_metric: true

Reviewer: ```suggestion
          create_metric: true
```

  | --- |
  | 2 |

#### Query filter

> You can also use metrics in filters at the query level. The same query as above can be generated by running this in the CLI:

Reviewer: ```suggestion
You can also use metrics in filters at the query level. The same query as above can be generated by running this in the command line interface (CLI):
```

> You can also use metrics in filters at the query level. The same query as above can be generated by running this in the command line interface (CLI):

Reviewer: we avoid using 'below/above' for accessibility purposes so tweaked this :

```suggestion
You can also use metrics in filters at the query level. Run this command in the command line interface (CLI) to generate the same SQL query referenced earlier:
```
